scss/_functions.scss

```diff
@@ -8,7 +8,7 @@
   $prev-key: null;
   $prev-num: null;
   @each $key, $num in $map {
-    @if $prev-num == null {
+    @if $prev-num == null or unit($num) == "%" {
```

Comment:
If I'm not mistaken, this only fixed % warning, if you use percentage at the biggest unit.

E.g. the bug report was: sm 500, md 700, lg 850, xl 100%.

But I have a use case, where I have only two categories: small: 80%, big: 720px.  So big is never too wide for reading, but small always has a nice padding from the screen (80% width).  This causes a warning still, because not the current unit is "%", but the previous unit.  Can we also ignore the warning in case of `unit($prev-num) == "%"`?

Reply:
@errge you should make a new issue